source/prefab/Card.h

```diff
@@ -0,0 +1,83 @@
+#ifndef EMP_CARD_INTERFACE_H
+#define EMP_CARD_INTERFACE_H
+
+#include "../web/Div.h"
+#include "../tools/string_utils.h"
+#include "Collapse.h"
+#include "FontAwesomeIcon.h"
+
+namespace emp {
+    namespace prefab{
```

Comment:
the double nesting makes everything get indented super far

emp {
prefab { 

proceed as usual